x/ecocredit/server/server_test.go

```diff
@@ -23,6 +23,18 @@ import (
 )
 
 func TestServer(t *testing.T) {
+	ff, ecocreditSubspace, bankKeeper, accountKeeper := setup(t)
+	s := testsuite.NewIntegrationTestSuite(ff, ecocreditSubspace, bankKeeper, accountKeeper)
+	suite.Run(t, s)
+}
+
+func TestInvalidGenesis(t *testing.T) {
```

Comment:
```suggestion
func TestGenesis(t *testing.T) {
```

should this just be `TestGenesis`? we could add more genesis tests later in which case the name wouldnt make sense